Sources/XcbeautifyLib/Renderers/TerminalRenderer.swift

```diff
@@ -140,11 +140,20 @@ struct TerminalRenderer: OutputRendering {
         return colored ? "\(Symbol.error) \(reason.f.Red)" : "\(Symbol.asciiError) \(reason)"
     }
 
-    // TODO: Print file path
     func formatLinkerDuplicateSymbolsError(group: LinkerDuplicateSymbolsCaptureGroup) -> String {
         let reason = group.reason
         return colored ? "\(Symbol.error) \(reason.f.Red)" : "\(Symbol.asciiError) \(reason)"
     }
+    
+    func formatLinkerDuplicateSymbolsLocation(group: LinkerDuplicateSymbolsLocationCaptureGroup) -> String? {
+        let filePath = group.filePath
+        return colored ? "   \(filePath.f.Yellow)" : "\(Symbol.asciiError) \(filePath)"
```

Comment:
I think this should use `Format.indent` instead of leading spaces.

Reply:
Got it.